mod/filter/lib/card/company_filter_query.rb

```diff
@@ -5,14 +5,17 @@ def company_wql company
     end
     alias wikirate_company_wql company_wql
 
-    def topic_wql value
-      add_to_wql :found_by, value.to_name.trait_name(:wikirate_company).trait(:refers_to)
+    def company_group_wql group
+      referred_to_by_company_list group
     end
-    alias wikirate_topic_wql topic_wql
```

Comment:
this didn't work at all.  there is no type_plus_right/topic/company content (they are linked via answers).  confirmed on live.